apps/package.json

```diff
@@ -130,7 +130,7 @@
     "newrelic": "^1.27.2",
     "node-sass": "1.2.2",
     "npm-which": "2.0.0",
-    "p5.play": "github:code-dot-org/p5.play#master",
+    "p5.play": "github:code-dot-org/p5.play#049c866cfc3d81f670361e8bf186a46cc9b6552b",
```

Comment:
When we push to our fork, we'll also need to update this commit number? Is there any reason we wouldn't want to point to master?